front-end/src/components/PostListItem.tsx

```diff
@@ -8,16 +8,14 @@ import PostDetailModal from './PostDetailModal';
 
 interface Props {
   post: Post;
-  // true if the user can edit this post
-  isEditable?: boolean;
-  // true if the user can delete the post
-  isDeletable?: boolean;
   loggedInUser?: UserLogin;
   removeFromFeed: Function;
 }
 
 export default function PostListItem(props:Props) {
 
+  const isAuthorPost: boolean = (props.loggedInUser ? props.loggedInUser.authorId : "") === props.post.author.id;
```

Comment:
Here's another way you can write this line using optional chaining and nullish coalescing.
I _think_ in the years to come this suggested way will be the preferred way to write it, but for now, it's probably non-standard and confusing.
This is more of a fun fact than a firm suggestion. No worries about writing it this way
```suggestion
  const isAuthorPost: boolean = props.loggedInUser?.authorId ?? "" === props.post.author.id;
```
NOTE: untested code

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator